definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-payment-by-account-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "02/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LabelSolStatementOTruthPara-4",
```

Comment:
You also need at least "R" permission for all the other roles: super-user, caseworker, etc

Reply:
Actually seems not needed for labels